docs/scanner-quickstart.md

```diff
@@ -0,0 +1,136 @@
+# Running a Scan Node
+
+!!! warning "Early Decentralization Phase"
+    At this phase, only approved nodes are allowed onto the network. As we introduce other decentralized security measures we will make this process more permissionless.
+
+## Scan Node Requirements
+
+The following are the requirements for running a Forta scan node.
+
+- 64-bit Linux distribution
+- CPU with 4+ cores
+- 16GB RAM
+- Connection to Internet
+- Docker v20.10+
+- **Recommended:** Ethereum Light Node
+
+## Install Forta
+
+The Forta scan node software is available for installing on popular 64-bit Linux distributions using official Forta repositories. Package installation methods are verifiable (auto-verified during installation) and help you install required dependencies.
+
+### Install via YUM (CentOS, Fedora, Red Hat Enterprise Linux etc.)
+
+```
+$ curl https://dist.forta.network/repositories/yum/Forta.repo -o /etc/yum.repos.d/Forta.repo -s
+$ yum install forta
+```
+
+### Install via APT (Ubuntu, Debian etc.)
+
+```
+$ wget https://dist.forta.network/pgp.public -O /usr/share/keyrings/forta-keyring.asc
+$ echo 'deb [signed-by=/usr/share/keyrings/forta-keyring.asc] https://dist.forta.network/repositories/apt stable main' > /etc/apt/sources.list.d/forta.list
+$ apt-get update
+$ apt-get install forta
+```
+
+### Install Manually
+
+1. [Install Docker](https://docs.docker.com/get-docker/) (at least v20.10) 
+2. Download the latest x86-64 release binary and install
+
+```
+$ wget https://dist.forta.network/artifacts/forta
+$ mv forta /usr/local/bin/forta
```

Comment:
Not a big deal but should we use `curl` as a "go-to" command for APT and manual instructions ( just like the yum instructions ) instead of `wget`, since `curl` comes by default (unless alpine) and for `wget` we need to `apt install` or `brew install` separately?
```suggestion
$ sudo curl https://dist.forta.network/artifacts/forta -o /usr/local/bin/forta
```